Sources/SpotHeroAPI/Search/Common/Models/FacilityOptions.swift

```diff
@@ -0,0 +1,13 @@
+// Copyright © 2021 SpotHero, Inc. All rights reserved.
+
+/// Contains information about misc options available to the user
+/// at a given parking facility based on search criteria and user information
```

Comment:
Adding a period just for consistency across other docs.
```suggestion
/// at a given parking facility based on search criteria and user information.
```